.eslintignore

```diff
@@ -1,2 +1,3 @@
 node_modules
 public/_js
+tmp*
```

Comment:
the custom firefox profile creates a temporary directory with JS that lab tries to lint so ignoring that